frontend/catalyst/python_interface/inspection/draw.py

```diff
@@ -51,8 +50,8 @@
 if which("dot") is None:
     HAS_GRAPHVIZ = False
 
-from ..visualization.construct_circuit_dag import ConstructCircuitDAG
-from ..visualization.pydot_dag_builder import PyDotDAGBuilder
+from .construct_circuit_dag import ConstructCircuitDAG
+from .pydot_dag_builder import PyDotDAGBuilder
```

Comment:
Have you tested what happens if pydot/graphviz aren't available and these files are imported?